cmd/openshift-install/create.go

```diff
@@ -376,7 +376,14 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 						cov1helpers.FindStatusCondition(cv.Status.Conditions, configv1.OperatorFailing).Message)
 					return false, nil
 				}
+				if progressingCond := cov1helpers.FindStatusCondition(cv.Status.Conditions, configv1.OperatorProgressing); progressingCond != nil {
+					if progressingCond.Message != lastProgressingMessage {
+						logrus.Info(progressingCond.Message)
+						lastProgressingMessage = progressingCond.Message
+					}
```

Comment:
I think you want `return false, nil` here to avoid the following debug log, since you've just provided more-detailed information here.